documentation/properties/leg_type.md

```diff
@@ -5,5 +5,9 @@ schemas:    [derivative]
 ---
 
 # leg_type
-Describe the payoff type of a derivative leg, which may be part of an instrument refered to in the derivative_type attibute (eg. vanilla_swap). The atribute is an enum with the following members:
-- fixed, floating, index, call, put
\ No newline at end of file
+Payoff type of a derivative leg, which may be a stand-alone trade (e.g. fra, cap), or part of an instrument refered to in the derivative_type attibute (eg. vanilla_swap). The atribute is an enum with the following members:
+- fixed: a leg with fixed amounts
+- floating: a leg in the interest rate asset class with variable interest amounts referencing an underlying_index (e.g. USD_LIBOR_BBA) and an optional underlying index tenor (e.g. "3m").
+- indexed: a leg in the fx, credit, equity and commodity asset class, with variable amounts which depend on the performance on an underlying security, issuer or index
```

Comment:
What is the difference between floating and indexed? Is there a more detailed description from the regs?
If my trade is 4% fixed for FTSE100 - 50bps, what is it?

Reply:
There is no description at all in the regs. 
your trade is indexed!
floating legs are used for floating interest payments referring to the underlying index ; the notional amount is fixed (not indexed).
indexed legs: are used for transactions where the notional amount is indexed to the underlying price variation/performance

Reply:
Ah, that makes more sense now when you talk about the notional. Thanks for clarifying!